assets/src/edit-story/components/library/panes/pageLayouts/pageLayout.js

```diff
@@ -53,42 +65,42 @@ PreviewPageWrapper.propTypes = {
   pageSize: PageSizePropType.isRequired,
 };
 
-const HoverControls = styled.div`
-  height: ${({ pageSize }) => pageSize.containerHeight}px;
-  width: ${({ pageSize }) => pageSize.width}px;
-  position: absolute;
-  opacity: ${({ isActive }) => (isActive ? 1 : 0)};
-  cursor: pointer;
-  border-radius: ${({ theme }) => theme.border.radius.default};
-  overflow: hidden;
-
-  ${KEYBOARD_USER_SELECTOR} &:focus {
-    outline: ${({ theme }) => `2px solid ${theme.colors.selection} !important`};
-  }
-`;
-HoverControls.propTypes = {
-  pageSize: PageSizePropType.isRequired,
-  isActive: PropTypes.bool.isRequired,
-};
-
 const PageLayoutTitle = styled.div`
   position: absolute;
   bottom: 0;
   background-color: ${({ theme }) => theme.colors.grayout};
+  border-radius: ${({ theme }) => theme.border.radius.default};
+  border-top-right-radius: 0;
+  border-top-left-radius: 0;
+  opacity: ${({ isActive }) => (isActive ? 1 : 0)};
+
   padding: 8px;
   font-size: 12px;
   line-height: 22px;
   width: 100%;
+  align-self: flex-end;
 `;
 
-function PageLayout(props) {
-  const { page, pageSize, onConfirm, requiresConfirmation } = props;
+PageLayoutTitle.propTypes = {
+  isActive: PropTypes.bool.isRequired,
+};
 
+function PageLayout(props) {
+  const {
+    page,
+    pageSize,
+    onConfirm,
+    requiresConfirmation,
+    translateY,
+    translateX,
+  } = props;
```

Comment:
Nit: I don't know what the code style is, but should we destructure these on line 88?

Reply:
🤷‍♀️ I was just keepin' with the patterning existing in the page layouts. You're right that descruturing is pretty common though, I can update this for continuity. I was trying to make as few updates to get accessibility working.

Reply:
Totally fine as is!

Reply:
I'd prefer moving it to the component definition unless you need the entire object for something later - e.g. if you'd pass on all props to some child component or similar.

Reply:
nah that sounds good, i'll move it. i was trying to stay focused on the arrow nav

Reply:
alright, moved 'em!